Sanity/DAST_test/main.fmf

```diff
@@ -0,0 +1,30 @@
+summary: DAST test for the tang operator
+description: ''
+contact: Martin Zelený <mzeleny@redhat.com>
```

Comment:
Please, lets take this opportunity to change the contact

Reply:
Sure, DONE.